src/release.ts

```diff
@@ -19,10 +19,30 @@ async function isWorktreeEmpty() {
   return !ret.stdout
 }
 
-export async function publish(preRelease: boolean | undefined) {
+interface PublishCommandOptions {
+  preRelease?: boolean
+  checkRemoteVersion?: boolean
+}
+export async function publish({ preRelease, checkRemoteVersion }: PublishCommandOptions) {
   const s = createSpinner('Publishing all packages').start()
   const args = ['-r', 'publish', '--no-git-checks', '--access', 'public']
 
+  if (checkRemoteVersion) {
+    const packageJson = getPackageJsons().find((packageJson) => !packageJson.config.private)
+    if (packageJson) {
+      const { config } = packageJson
+      try {
+        await execa('npm', ['view', `${config.name}@${config.version}`, 'version'])
+        s.warn({
+          text: `The npm package has a same remote version ${config.version}, please check it https://www.npmjs.com/package/${config.name}?activeTab=versions`,
```

Comment:
The text change to `The npm package has a same remote version ${config.version}, publishing automatically skipped`